packages/svelte/src/compiler/phases/3-transform/server/transform-server.js

```diff
@@ -1845,6 +1846,7 @@ function serialize_element_attributes(node, context) {
 	// Use the index to keep the attributes order which is important for spreading
 	let class_attribute_idx = -1;
 	let style_attribute_idx = -1;
+	let events_to_capture = [];
```

Comment:
this should be a `Set`, otherwise this...

```svelte
<img alt="blah" src="blah" onload={onload} {...{}} use:blah>
```

...yields this:

```js
$$payload.out += `<img${$.spread_attributes([{ "alt": "blah" }, { "src": "blah" }, {}], true, true, "")} onload="this.__e=event" onload="this.__e=event" onerror="this.__e=event" onload="this.__e=event" onerror="this.__e=event">`;
```

(side-note, what the hell is going on with `spread_attributes` — will open a separate issue for that)